sentry/templates/configmap-sentry.yaml

```diff
@@ -468,51 +406,76 @@ data:
     SENTRY_OPTIONS['mail.host'] = os.getenv("SENTRY_EMAIL_HOST", {{ .Values.mail.host | quote }})
     SENTRY_OPTIONS['mail.from'] = os.getenv("SENTRY_EMAIL_FROM", {{ .Values.mail.from | quote }})
 
+    #######################
+    # MaxMind Integration #
+    #######################
+
+    {{- if .Values.geodata.path }}
+    GEOIP_PATH_MMDB = {{ .Values.geodata.path | quote }}
+    {{- end }}
+    
     #########################
     # Bitbucket Integration #
     #########################
-
+    
     # BITBUCKET_CONSUMER_KEY = 'YOUR_BITBUCKET_CONSUMER_KEY'
     # BITBUCKET_CONSUMER_SECRET = 'YOUR_BITBUCKET_CONSUMER_SECRET'
 
-    #########
-    # Relay #
-    #########
-    SENTRY_RELAY_WHITELIST_PK = []
-    SENTRY_RELAY_OPEN_REGISTRATION = True
-
-    #######################
-    # OpenAi Suggestions #
-    #######################
+    ##############################################
```

Comment:
updated to match self-hosted config for openai

[reference](https://github.com/getsentry/self-hosted/blob/master/sentry/sentry.conf.example.py#L296C1-L296C1)